src/cluster/cluster.ts

```diff
@@ -1,17 +1,16 @@
-"use strict";
+/// <reference path="../../typings/clusterfck.d.ts"/>
```

Comment:
@yhoonkim  For JS-based library dependency, need to declare type file and make reference like this.